src/App.js

```diff
@@ -81,8 +91,17 @@ const App = () => {
           />
         ))}
       </div>
+      <hr/>
+      <div>
+        {productList.products.map((data) => 
```

Comment:
Code is not indented properly. Can you at-least indent the code before submitting?